bigbluebutton-html5/imports/ui/components/user-list/service.js

```diff
@@ -320,6 +320,20 @@ const isMeetingLocked = (id) => {
   return isLocked;
 };
 
+const isWaitingApprovalEnabled = (meetingId) => {
+  let isEnabled = false;
+  const meeting = Meetings.findOne(
+    { meetingId: meetingId },
+    { fields: { 'usersProp.guestPolicy': 1 } },
+  );
+
+  if (meeting.usersProp.guestPolicy === 'ASK_MODERATOR') {
+    isEnabled=true;
+  };
+
+  return isEnabled ;
+};
+
```

Comment:
`getAvailableActions` receives the `usersProp` as param. This is made to avoid doing multiple queries to the `Meeting` collection just like this one. Along with the `guestPolicy`, `usersProp` also has the `authenticatedGuest` value (the same one we set at bbb-web). When this `authenticatedGuest` is false, the only users that can be moved back to the lobby should be the ones that have `guest=true` (including moderators)